spec.html

```diff
@@ -39453,6 +39235,20 @@ <h1>AsyncGeneratorYield ( _value_ )</h1>
           1. NOTE: This returns to the evaluation of the operation that had most previously resumed evaluation of _genContext_.
         </emu-alg>
       </emu-clause>
+
+      <emu-clause id="sec-createasynciteratorfromclosure" aoid="CreateAsyncIteratorFromClosure">
+        <h1>CreateAsyncIteratorFromClosure ( _closure_, _generatorBrand_, _generatorPrototype_ )</h1>
+        <p>The abstract operation CreateAsyncIteratorFromClosure takes arguments _closure_ (an Abstract Closure with no parameters), _generatorBrand_, and _generatorPrototype_ (an Object). It performs the following steps when called:</p>
+        <emu-alg>
+          1. NOTE: _closure_ can contain uses of the Await shorthand and uses of the Yield shorthand to yield an IteratorResult object.
+          1. Let _internalSlotsList_ be &laquo; [[AsyncGeneratorState]], [[AsyncGeneratorContext]], [[AsyncGeneratorQueue]], [[GeneratorBrand]] &raquo;.
+          1. Let _generator_ be ! OrdinaryObjectCreate(_generatorPrototype_, _internalSlotsList_).
+          1. Set _generator_.[[GeneratorBrand]] to _generatorBrand_.
+          1. Set _generator_.[[AsyncGeneratorState]] to *undefined*.
+          1. Perform ? AsyncGeneratorStart(_generator_, _closure_).
```

Comment:
```suggestion
          1. Perform ! AsyncGeneratorStart(_generator_, _closure_).
```
(I don't think AsyncGeneratorStart can return an abrupt completion.)